code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -916,20 +916,18 @@ var/list/datum/dna/hivemind_bank = list()
 
 /mob/proc/changeling_prepare_DEATHsting()
 	set category = "Changeling"
-	set name = "Death Sting (40)"
+	set name = "Cryo Sting (50)"
 	set desc = "Causes spasms onto death."
 
-	check_CH("Death Sting",/datum/click_handler/changeling/changeling_DEATHsting)
+	check_CH("Cryo Sting",/datum/click_handler/changeling/changeling_DEATHsting)
 	return
 
 /mob/proc/changeling_DEATHsting(atom/A)
-	var/mob/living/carbon/T = changeling_sting(40,A)
+	var/mob/living/carbon/T = changeling_sting(50,A)
 	if(!T)	return 0
-	T << SPAN_DANGER("You feel a small prick and your chest becomes tight.")
-	T.silent = 10
-	T.Paralyse(10)
-	T.make_jittery(1000)
-	if(T.reagents)	T.reagents.add_reagent("lexorin", 40)
+	T << SPAN_DANGER("You feel a small prick and your blood begins to freeze in your veins.")
```

Comment:
These should be `to_chat()`

Reply:
> These should be `to_chat()`

The reason it uses SPAN_DANGER is to make it a warning text that only appears on the target's client, complete with italic font and red coloring.

Reply:
The reason why we changing from << to to_chat is to avoid hardcoded lines like this shit. Basically T << SPAN_DANGER( is same as to_chat( T , SPAN_DANGER(.....))